web/styles/input_pill.css

```diff
@@ -284,6 +284,10 @@
     }
 }
 
+.display_only_group_pill .zulip-icon-triple-users {
+    font-size: 19px;
```

Comment:
Should this be in `em` to scale properly with font size?

Reply:
I tried using em value of `1.19em` (computed from using 19px for 16px font size), but feels like 19px is better visually -
| | Using 19px | Using 1.19em |
| ----------- | --------------- | ------------------ |
| For 14/122px | ![compact-19px](https://github.com/user-attachments/assets/ffd4f699-d813-4039-98b9-cf48cce5bc9a) | ![compact-em](https://github.com/user-attachments/assets/4f4542ef-481f-45b8-a542-6876da536559) |
| For 16/140px | ![19px](https://github.com/user-attachments/assets/f27b2762-f801-4b30-8b50-923ea15750d6) | ![em](https://github.com/user-attachments/assets/e1f897c7-696b-488a-8022-5ddca2ed1f04) |